app/src/components/homePageMainContent/homePageMainContent.tsx

```diff
@@ -1,32 +1,53 @@
 import React from "react";
 import FeaturedContentComponent from "../featuredContent/featuredContent";
 import { useEffect, useState } from "react";
-import CollectionLanes from "../lanes/collectionLanes/collectionLanes";
-import CollectionLanesLoading from "../lanes/collectionLanes/collectionLanesLoading";
 import { DC_URL } from "@/src/config/constants";
+import { Lane as DCLane } from "../lane/lane";
+import LaneDataType from "@/src/types/Lane";
+import LaneLoading from "../lane/laneLoading";
 
-const HomePageMainContent = ({ data }) => {
+interface HomePageMainContentProps {
+  swimlanes: LaneDataType[];
+  randomNumber: number;
+}
+
+const HomePageMainContent = ({
+  swimlanes,
+  randomNumber,
+}: HomePageMainContentProps) => {
   const [isLoaded, setIsLoaded] = useState(false);
 
   useEffect(() => {
     setIsLoaded(true);
   }, []);
 
+  const firstSwimLane = swimlanes[0];
   return isLoaded ? (
     <>
-      <CollectionLanes
-        lanesWithNumItems={[data.lanesWithNumItems[0]]}
-        seeMoreLink={`${DC_URL}/collections/lane`}
-      />
-      <FeaturedContentComponent randomNumber={data.randomNumber} />
-      <CollectionLanes
-        lanesWithNumItems={data.lanesWithNumItems.slice(1)}
+      <div style={{ marginTop: "-3rem" }} />
```

Comment:
Why did you have to add this?

Reply:
The `Lane` component has a top margin which is needed in all cases except the first lane on the homepage, which needs to be closer to the hero. Since now the lane component is represented as a singular lane, which the parent component replicates as needed, I think the parent component should also be responsible for removing the margin on the first child (instead of the `Lane`s, plural, innately knowing the first one needs to not have a margin). I could do this by declaring a separate CSS module with a first child rule, or just do this.